config/mainnet.dhall

```diff
@@ -0,0 +1,5 @@
+{ keyType      = ""
+, reportServer = "http://report-server.cardano-mainnet.iohk.io:8080"
+, updateServer = "http://update.cardano-mainnet.iohk.io"
```

Comment:
This url is now changed in existing launcher files

Reply:
Yes, this PR's base has aged, need to adjust..

Reply:
Rebased on top of current `release/0.9.0`